Azure/azure-cli
# [Find] Fix #14094 az find Fix Queries failing when not logged in and when telemetry is disabled

**Description**  

The Aladdin service expects to receive a GUID for both the `correlationId` and `subscriptionId` if they are sent. If users were not logged in or had disabled their telemetry, this service would return a 400, since the values passed in were considered strings, not GUIDs. The error message from the server is no longer displayed. It is now a more user friendly message.

I believe #14094 was caused by this issue, though the user was unable to reproduce it, so there could be other factors at play.

**Testing Guide**  

`az find "create a vm"`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
